releasenotes/notes/fix-generalpm-bugs-ed901004647c39ee.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixed missing :attr:`.~GenericPass.property_set` in the :class:`.GenericPass` base class.
+  - |
+    Fixed the bug that :class:`.DoWhileController` iterates over
+    the tasks of :class:`.GenericPass` subclass instance without executing them.
+  - |
+    Fixed the bug that :class:`.BasePassManager` accidentally splits the input value at run
+    when the pass manager input is a python string.
```

Comment:
If we merge this into 0.45.0 we don't actually need a release note (because the buggy behaviour was never released), though if it waits til 0.45.1 we will need one.